app/cdap/components/WrangleHome/Components/OngoingDataExplorations/utils.ts

```diff
@@ -15,42 +15,47 @@
  */
 
 import { ImportDatasetIcon } from '../WrangleCard/iconStore/ImportDatasetIcon';
-import { IMassagedObject } from './types';
+import { IMassagedObject, IEachData } from './types';
 
-export const generateDataForExplorationCard = (oldData) => {
+export const generateDataForExplorationCard = (oldData: IEachData[], cardCount: number) => {
```

Comment:
modify oldData to some appropriate name like existingOngoingExplorationsData

Reply:
modify the name of the type for oldData as well.( IExistingExplorationCardsData[] )